site/en/remote/output-directories.md

```diff
@@ -67,8 +67,8 @@ The directories are laid out as follows:
 
 <pre>
 &lt;workspace-name&gt;/                         <== The workspace root
-  bazel-my-project => <...my-project>     <== Symlink to execRoot
-  bazel-out => <...bin>                   <== Convenience symlink to outputPath
+  bazel-my-project => <..._main>          <== Symlink to execRoot
```

Comment:
Does this mean `bazel-my-project` is not longer a symlink to execroot? My bazel directory shows it pointing to `.../execroot/_main`, which is what you're expressing here?

CC @Wyverald because of linked related changes.

Reply:
> Does this mean bazel-my-project is not longer a symlink to execroot? My bazel directory shows it pointing to .../execroot/_main, which is what you're expressing here?

It is a symlink to "execRoot" - i.e. the place where actions are actually executed, or, in case of sandbox, in place that looks like execRoot. Before ~bazel 7 actions were executed in directory 
`.../<workspace_name>` and right now are in `.../_main` no matter what the worskpace name is. For example:

```
$ cat BUILD
genrule(
    name="check_pwd",
    cmd_bash="pwd",
    outs=["missing"],
)

$ bazel build :check_pwd --spawn_strategy=local
/home/hauser/.cache/bazel/_bazel_hauser/5a7666258fa52d69749438b55b3c5a01/execroot/_main
```